Example/NYTPhotoViewer-Swift/PhotosProvider.swift

```diff
@@ -29,7 +29,15 @@ class PhotosProvider: NSObject {
         
         for photoIndex in 0 ..< NumberOfPhotos {
             let title = NSAttributedString(string: "\(photoIndex + 1)", attributes: [NSForegroundColorAttributeName: UIColor.whiteColor()])
-            let photo = shouldSetImageOnIndex(photoIndex) ? ExamplePhoto(image: image, attributedCaptionTitle: title) : ExamplePhoto(attributedCaptionTitle: title)
+            
+            let imageData: NSData? = {
+                if let image = image {
+                    return UIImagePNGRepresentation(image)
+                }
+                return nil
+            }()
+            
+            let photo = shouldSetImageOnIndex(photoIndex) ? ExamplePhoto(imageData: imageData, attributedCaptionTitle: title) : ExamplePhoto(attributedCaptionTitle: title)
```

Comment:
Since we're not using the animated image functionality, would it make more sense and be more instructive int he example project to instead initialize an `ExamplePhoto` with a `UIImage` directly instead of `NSData`? It seems like the more common use case, and this example may be a bit confusing for newcomers as to why they would have to call `UIImagePNGRepresentation` to get the image data in their model object.